### PR TITLE
Issue #241 Fixed all footer issues

### PR DESCRIPTION
**Pull request by Albert-Jokelin**
- [x] Halved the footer grid-gap; from 4em to 2em.
- [x] Fixed the alignment issues in footer elements
- [x] Increased spacing in between elements in the footer.

Check it out: https://suspicious-clarke-a869e9.netlify.app/

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
Screenshot (174)

There is a large gap in top, decrease it.
And add a little padding below the copyright line.

File: Home.html [hunk 1]
```diff
@@ -136,13 +136,13 @@
...
   <!-- navbar starts -->
   <header>
-    <div class="navbar"> 
+    <div class="navbar">
       <div class="main-heading">
 
         <div class="logo">
           <a href="#" class="endlogo">
             <img src="./Images/flag.png">
-
```
Comment (nlok5923) [inline Home.html:145]: @Albert-Jokelin there are so many newline added instead of previous trailing spaces please revert this

File: help.css [hunk 2]
```diff
@@ -726,8 +734,9 @@ tr td{
   color: black;
 }
 
-@media(max-width: 1240px) {
-  #footer {
+@media (max-width: 1240px) {
+
```
Comment (nlok5923) [inline help.css:738]: unnecessary newline

**Review by nlok5923 — CHANGES_REQUESTED** [review]
PTAL

**Comment by Albert-Jokelin** [comment]
I'm sorry, I got caught up with something. Will make changes right away

**Comment by QAZIMAAZARSHAD** [comment]
@Albert-Jokelin always mention issue number like this #241 in PR description.

**Pull request merged**